products/compute/api.yaml

```diff
@@ -15400,10 +15402,13 @@ objects:
                           original URL is retained. Defaults to false.
             - !ruby/object:Api::Type::NestedObject
               name: 'defaultUrlRedirect'
-              # TODO: add defaultRouteAction.weightedBackendService here once they are supported.
-              exactly_one_of:
-                - path_matchers.0.default_service
-                - path_matchers.0.default_url_redirect
+              # TODO: (mbang) won't work for array path matchers yet, uncomment here once they are supported.
+              # exactly_one_of:
+              #   - path_matchers.0.default_service
+              #   - path_matchers.0.default_url_redirect
+              #   - path_matchers.0.default_route_action.0.weighted_backend_services
+              conflicts:
```

Comment:
does this also conflict with `default_service`? it seems like it would based on the `exactly_one_of` but the doc comment for `default_route_action` also sounds like it's just `default_url_redirect`

Reply:
I went through and tried to figure out exactly how they all correlate. I've updated it now.